app/js/pluginManager.js

```diff
@@ -135,6 +129,16 @@ function PluginManager() {
 					event.args[1].left += $('#sidebar').width();
 					UI.tooltip.apply(null, event.args);
 					break;
+				case 'config':
+					// get or set something in the config.json
+					// TODO: Document in PluginTutorial.md
+					var args = event.args[0];
+					var responseChannel = event.args[1];
+					var result = UI.config(args);
+					if (responseChannel) {
+						plugin.sendToView(responseChannel, result)
```

Comment:
if you were to insert another line here, the resulting parse tree would be unpredictable. I believe JS now automatically inserts semicolons in certain scenarios, but I'd rather not risk it.